pkgs/development/embedded/arduino/arduino-language-server/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
+
+{
+  pkg = buildGoModule rec {
```

Comment:
have you ever read the manuals?

Reply:
Which one?

I'm assuming I did something wrong here but I have no idea what.